CHANGELOG.md

```diff
@@ -0,0 +1,3 @@
+- Deploying a function with a retry policy will no longer prompt if the function already had a retry policy
+- Fixes issue where the Firebase Hosting emulator would fail to start with `--only` filters using targets (#2820).
+- Adds `hosting:sites:<create|get|list|delete>` commands to allow managing Hosting Sites.
```

Comment:
This feels like it's adding too much

Reply:
Oops, merge conflict.  Removed the extra lines.